packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -27,6 +34,40 @@ function getTokenAddressOrThrow(assetData: string): string {
     throw new Error(`Decoded asset data (${JSON.stringify(decodedAssetData)}) does not contain a token address`);
 }
 
+function assertTakerAddressOrThrow(takerAddress: string | undefined): void {
+    if (
+        takerAddress === undefined ||
+        takerAddress === '' ||
```

Comment:
```suggestion
        !takerAddress ||
```

Reply:
I made this change here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/83289bc801013e1b7d2bc0b6f5696d886be90784